### PR TITLE
#153071337 Build Object Oriented classes and test them

### PR DESCRIPTION
**Pull request by betsybeth**
#### the main directory has the following modules:
* person.py
* user.py
* event.py
* rsvp.py
#####  the test directory has the following modules:
* test_person.py
* test_user.py
* test_event.py
* test_rsvp.py
##### To run tests cd into Bright_event Run `nosetests -v`
@Mnickii

### CODE REVIEW

**Comment by Rodgers-M** [comment]
good PR, follows best practices conventions. nice arrangement of code.

File: app/tests/test_event.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 """ modules and libraries used for Test Event class """
 
 import unittest
-from main.event import Event
-from main.rsvp import Rsvp
+from ..main.event import Event
```
Comment (Mnickii) [inline app/tests/test_event.py]: Good implementation of feedback

**Comment by Mnickii** [comment]
You can now merge and continue

**Comment by Gidraff** [comment]
Good work

**Comment by Gidraff** [comment]
This is really good.Great improvement Beth

**Comment by ngeno7** [comment]
Good work the project structure.

**Comment by frankip** [comment]
good work on the test coverage

**Pull request closed without merging**